modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+* When creating a space, the creator is automatically in the delegatable Manager role for this space. For details see xref:space-roles-and-permissions[Space Roles and Permissions].
+
+Admin::
+The `Admin` role is in full control of all permission types with one exception which is `Self Management`. This permission type is excluded to avoid that the last admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set by the `Role Management` permission.
+
+Space Admin::
+The `Space Admin` role allows you to create spaces, set quotas and delegating the right to manage space members and space member roles.
+
+User::
+The `User` role can manage his own account and change his language. To manage space group members and space member roles, an Admin or Space Admin must create one and grant the Manager permission to the user.
+
+Guest::
+Users with the `Guest` role assigned can only change their language.
+
+== Space Roles and Permissions
+
+// https://github.com/owncloud/web/blob/3ad8561a1c7ac5ea4f1b30397569f4b9c18166f7/packages/web-pkg/src/services/permissionManager.ts
+
+When it comes to permissions for a space, three roles are available:
+
+* Manager
+* Editor
+* Viewer
+
+Note that the creator of a space becomes manager automatically, can add other users and delegate his Manager role to one or more space members.
+
+The following table describes the Space Roles and it's permissions. Note that permissions are added as higher the role gets.
+
+[width=60,cols=".^30%,100%",options="header",]
+|===
+| Role
+| Permissions
+
+| Viewer
+a| * read documents,
+* view images,
+* play videos,
+* get info about the space.
+
+| Editor
+a| * modify documents and even delete them,
+* upload content to a space,
+* edit the space name,
+* edit the space description,
+* add or change the image for the space,
+* empty the trash bin of the space.
+
+| Manager
+a| * invite additional members,
+* share a space,
+* deactivate and activate a space ^(1)^,
+* delete a space
+* change the roles of members, +
+including those of other managers. A downgrade to viewer or editor is possible. 
+|===
+
+(1) ... With deactivating, a space Manager can disable any access to a space including sharing and syncing. This can be used to keep data in a frozen state before finally deleting it later on. To regain full access including sharing and syncing, a deactivated space can be activated again.
+
+=== Special Cases
+
+When a space has only one user with the Manager role, it may happen that the space becomes unmanaged. Possibly because the user does not have access anymore like when leaving the company or when serious health conditions occur making any change impossible. To solve this issue and for the time being, you must incorporate that user at your _external_ IDP side and delegate the Manager role to another space member post login.
```

Comment:
```suggestion
When a space has only one user with the Manager role, it may happen that the space becomes unmanaged. Possibly because the user with that role leaves the company or suffers serious health issues. To solve this problem for the time being, you must incorporate that user at your _external_ IDP side and delegate the Manager role to another space member post login.
```